larray/core.py

```diff
@@ -3999,7 +4013,26 @@ def __setattr__(self, key, value):
 
     def __dir__(self):
         names = set(axis.name for axis in self.axes if axis.name is not None)
-        return list(set(dir(self.__class__)) | names)
+        attrs = set(self._attrs.keys())
```

Comment:
attrs should not appear in `LArray.__dir__` (since they are not available as arr.xyz), but in `AttrDict.__dir__`